src/lightning/pytorch/accelerators/accelerator.py

```diff
@@ -45,3 +45,8 @@ def get_device_stats(self, device: _DEVICE) -> dict[str, Any]:
 
         """
         raise NotImplementedError
+
+    @classmethod
+    def device_name(cls, device: Optional[_DEVICE] = None) -> str:
```

Comment:
this is not a classmethod. A classmethod is returning an instance of the class. This would be a static method